src/System.Net.Sockets.Legacy/tests/PerformanceTests/project.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": {
+    "System.Collections": "4.0.0-beta-*",
+    "System.Diagnostics.Debug": "4.0.10-beta-*",
+    "System.Runtime": "4.0.20-beta-*",
+    "System.Threading": "4.0.0-beta-*",
+    "System.Threading.Tasks": "4.0.0-beta-*",
+    "System.Threading.Tasks.Parallel":  "4.0.0-beta-*",
```

Comment:
@ericstj can confirm, but I believe we should be removing the "-beta-*" from most of these (same in the other project.json files)